### PR TITLE
update ledger-go to 0.12.2

### LINKED ISSUE #14: Upgrade to latest ledger-go

### PR DESCRIPTION
**Pull request by rllola**
closes #14

### CODE REVIEW

File: user_app.go [hunk 1]
```diff
@@ -20,7 +20,7 @@ import (
 	"errors"
 	"math"
 
-	"github.com/cosmos/ledger-go"
+	"github.com/zondax/ledger-go"
```
Comment (marbar3778) [inline user_app.go:23]: should people be importing cosmos or zondax here?

**Review by marbar3778 — APPROVED** [review]
utACK

File: user_app.go (hunk 1 above)
Comment (jleni) [reply user_app.go:23]: `ledger-go` is a library that we actually use for a large amount of projects. It is 100% generic and allows connecting to ledger devices from golang.

We maintain that library much more frequently. If you agree, I would keep this as Zondax.
Would that be fine?

Comment (marbar3778) [reply user_app.go:23]: oh amazing!!

**Pull request merged**